test/Fail/ImportInMutual.err

```diff
@@ -1,2 +1,13 @@
 ImportInMutual.agda:7,10-21
 Import statements are not allowed in mutual blocks
+ImportInMutual.agda:7,10-21
+Failed to find source of module Fake.Module in any of the following
```

Comment:
This does not look like the right error message.

Reply:
This was fixed in 3eb01a43bf17a0881309768c03b09402e587ac74: in this PR I forgot to actually drop the dead code in mutual blocks.

Actually removing it leads to `ImportInMutual` being moved to `Success`: https://github.com/agda/agda/commit/3eb01a43bf17a0881309768c03b09402e587ac74#diff-2e24d54ee33b98b733edeaf3e5e3cc14